packages/ui/src/helpers/authenticator/translations.ts

```diff
@@ -0,0 +1,139 @@
+import {
+  AuthChallengeName,
+  CodeDeliveryDetails,
+  SocialProvider,
+} from '../../types';
+import { translate } from '../../i18n';
+import { AuthenticatorRoute } from './facade';
+
+/**
+ * ConfirmSignIn
+ */
+const getChallengeText = (challengeName: AuthChallengeName): string => {
```

Comment:
Here are all the text util functions that centralizes and translates Authenticator texts. React Authenticator is used as source of truth.